lsstseries/ensemble.py

```diff
@@ -434,3 +452,47 @@ def _build_index(self, obj_id, band):
         tuples = zip(obj_id, band, idx)
         index = pd.MultiIndex.from_tuples(tuples, names=["object_id", "band", "index"])
         return index
+
+    def sf2(self, band_to_calc=None, combine=False,
+            method="size", sthresh=100):
```

Comment:
What happens if user passes a lightcurve that has less than 100 pairs?

Reply:
Whenever sthresh exceeds the total number of pairs, it simply lumps all pairs into a single bin. This should be true for all three methods.

Reply:
I suggest adding a warning that the number of pairs is lower than the treshold - either lower the threshold or increase the number of data points.

Reply:
My concern with that is that warnings won't scale, e.g. if we batch run 1 billion objects and even 1000 have this issue the output warnings are just spam and not useful